### PR TITLE
dashboard style guide

### LINKED ISSUE #168: Integrate Style Guide - Admin & General User Dashboard
- Integrate style guide for buttons

### PR DESCRIPTION
**Pull request by ctc-devops**
- Integrate style guide for buttons
closes #168

### CODE REVIEW

**Review by aliu55 — DISMISSED** [review]
LGTM

File: src/components/MarkerInfo/DeleteBoxModal/DeleteBoxModalContent.jsx [hunk 1]
```diff
@@ -10,7 +10,6 @@ const DeleteBoxModalContent = ({ deleteBox, deleteTransaction }) => (
       Do you want to delete the entire box or just the last transaction?
     </div>
     <div className={styles.buttons}>
```
Comment (meghakak) [inline src/components/MarkerInfo/DeleteBoxModal/DeleteBoxModalContent.jsx:12]: Please change these buttons to use the colorScheme from ChakraTheme.jsx

**Review by meghakak — CHANGES_REQUESTED** [review]
Very cool :) There are a few buttons throughout the dashboard that should be using the ChakraTheme colorSchemes instead of what they are currently using including: Undo in RelocationBox, Delete buttons, Review Submission, Admin Login, etc. Please go through these buttons and change them to use the ChakraTheme colorSchemes or add new ones as needed.

File: src/components/MarkerInfo/DeleteBoxModal/DeleteBoxModalContent.jsx (hunk 1 above)
Comment (claudeisakeeb) [reply src/components/MarkerInfo/DeleteBoxModal/DeleteBoxModalContent.jsx:12]: yes

File: src/components/PickupBox/PickupBox.jsx [hunk 2]
```diff
@@ -196,7 +196,11 @@ const PickupBox = ({
                           <button type="button" className={styles['approval-button']}>
                             <BsFillCheckCircleFill color="green" />
```
Comment (meghakak) [inline src/components/PickupBox/PickupBox.jsx]: Please use the vars.css colors here

File: src/components/PickupBox/PickupBox.jsx [hunk 3]
```diff
@@ -205,26 +209,50 @@ const PickupBox = ({
                           <button type="button" className={styles['rejection-button']}>
                             <BsXCircleFill color="red" />
```
Comment (meghakak) [inline src/components/PickupBox/PickupBox.jsx]: Please use the vars.css colors here

File: src/components/PickupBox/PickupBox.jsx [hunk 4]
```diff
@@ -205,26 +209,50 @@ const PickupBox = ({
...
-                    </button>
+                    {imageStatus !== 'rejected' && imageStatus !== 'approved' && picture && (
+                      <>
+                        {/* Approve image button */}
+                        <button
+                          type="button"
+                          className={styles['image-approved-button']}
+                          onClick={async () => updateImageStatus('approved')}
+                        >
+                          <BsFillCheckCircleFill color="green" />
```
Comment (meghakak) [inline src/components/PickupBox/PickupBox.jsx]: Same here

File: src/components/PickupBox/PickupBox.jsx [hunk 5]
```diff
@@ -205,26 +209,50 @@ const PickupBox = ({
...
+                        </button>
+                        {/* Reject image button */}
+                        <button
+                          type="button"
+                          className={styles['image-rejected-button']}
+                          onClick={async () => updateImageStatus('rejected')}
+                        >
+                          <BsXCircleFill color="red" />
```
Comment (meghakak) [inline src/components/PickupBox/PickupBox.jsx]: And here

**Review by meghakak — CHANGES_REQUESTED** [review]
Couple more color changes and this should be good to go

**Review by meghakak — APPROVED** [review]
LGTM

**Review by aliu55 — APPROVED** [review]
LGTM

**Pull request merged**